source/blog/2016-07-29-sprint-43-review-july-13-2016.html.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Sprint 43 review - July 13, 2016
+author: cybette
+date: 2016-07-29 14:38:08 UTC
+comments: true
+published: true
+---
+
+There is a ManageIQ Sprint review every 3 weeks, and everyone in the community is welcome to participate. To join our Sprint review meeting, please use this [Bluejeans link](https://bluejeans.com/946365937/).
+
+For Sprint 43, we merged 329 pull requests, making this a regular sprint averaging 100 PRs per week. There is a good distribution between new features, bug fixes, and code refactoring. We have plenty of enhancements on the Providers side, including Amazon, Networking, and Hawkular.  Work has started on Service Broker, which is a new piece for ManageIQ to allow VM selection based on criteria you provide.  
+
+SmartState has been extended with a generalized disk LRU caching module, which will be employed to address Azure performance and throttling issues. PostgreSQL has been upgraded to 9.5. There are also many User Interface updates, explained in detail in the [following video recording](https://www.youtube.com/watch?v=AnD1lvzsjB0) of the meeting, including a couple of demos in the Providers area (Recheck Authentication and Credentials Validation).
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/AnD1lvzsjB0" frameborder="0" allowfullscreen></iframe>
+
+As we had audio issues during this meeting with people dropping in and out, some of the presentations were not in the order laid out in the agenda. Here is the actual order with the corresponding timestamps for each section: 
+
+* Sprint Statistics (Oleg Barenboim) - 0:53
```

Comment:
Optional, but you can link these points in the video directly

```markdown
* Sprint Statistics (Oleg Barenboim) - [0:53](https://youtu.be/AnD1lvzsjB0?t=53)
```

renders to

* Sprint Statistics (Oleg Barenboim) - [0:53](https://youtu.be/AnD1lvzsjB0?t=53)